### ISSUE TITLE
ModView null messages from mod bug crash

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#1006]
# Proposed change
- There is a bug where a mod can send a message and the message is null(problem with parsing). 
- If the message is clicked it will crash the app

# Why is this important
- This is a main feature and it needs to be completed before the next release is sent out 

# Additional context
- the crash is caused by this code
```
 val messages = listChats.filter {
            it.displayName == clickedUsername }
            .map { if (it.deleted)  it.userType!! + " (deleted by mod)" else it.userType!!   }

```
- this code is inside of StreamViewModel.updateClickedChat()
- obviously `it.userType!!` needs a null check

**Comment by thePlebDev** [comment]
- I think this null message is from a sub message

**Comment by thePlebDev** [comment]
- closed with #1012
